website/source/layouts/docs.erb

```diff
@@ -465,28 +465,48 @@
       </li>
 
       <li<%= sidebar_current("docs-connect") %>>
-        <a href="/docs/connect/index.html">Connect</a>
+        <a href="/docs/connect/index.html">Connect - Service Mesh</a>
         <ul class="nav">
           <li<%= sidebar_current("docs-connect-config") %>>
             <a href="/docs/connect/configuration.html">Configuration</a>
           </li>
+          <li<%= sidebar_current("docs-connect-observability") %>>
+            <a href="/docs/connect/observability.html">Observability</a>
+          </li>
+        </li>
+        <li<%= sidebar_current("docs-connect-intentions") %>>
+          <a href="/docs/connect/intentions.html">Intentions - Security</a>
+        </li>
+        <li<%= sidebar_current("docs-connect-internals") %>>
+          <a href="/docs/connect/connect-internals.html">Architecture</a>
+        </li>
           <li<%= sidebar_current("docs-connect-proxies") %>>
-            <a href="/docs/connect/proxies.html">Proxies</a>
+            <a href="/docs/connect/proxies.html">Proxy Choice</a>
```

Comment:
How would you feel about calling this "Supported Proxies"? That implies there is a choice and lets the user know immediately that there will be a list of proxies they should be using (even if its just Envoy for now).